app.rb

```diff
@@ -1,9 +1,36 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
 
+  get '/newsfeed' do
+    @peeps = Peep.all
+    erb(:newsfeed)
+  end
+
+  post '/newsfeed' do
```

Comment:
If this is creating a new peep, I expect that the route would be **post /peep** to show you're creating a new peep rather than **post /newsfeed** which might confuse someone that you're creating a newsfeed resource (whatever that might look like).